dragula.js

```diff
@@ -301,6 +305,17 @@ function dragula (initialContainers, options) {
     var item = _copy || _item;
     var elementBehindCursor = getElementBehindPoint(_mirror, _clientX, _clientY);
     var dropTarget = findDropTarget(elementBehindCursor, _clientX, _clientY);
+    if (dropTarget !== null && !_emittedOver) {
+        _lastTargetOver = dropTarget;
+        _emittedOver = true;
+        // trigger over
+        api.emit('over', item, dropTarget, _source);
+    }
+    else if (dropTarget === null && _emittedOver) {
```

Comment:
please stay with the existing coding style, move the `else` to the previous line